core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/orchestration/ClusterTerminationHandler.java

```diff
@@ -35,8 +31,7 @@ public void accept(Event<ClusterTerminationRequest> event) {
         ClusterTerminationRequest request = event.getData();
         ClusterTerminationResult result;
         try {
-            Boolean allowed = clusterTerminationService.deleteClusterComponents(request.getClusterId());
-            result = new ClusterTerminationResult(request, allowed);
+            result = new ClusterTerminationResult(request, Boolean.TRUE);
```

Comment:
Can you add Deprecated or some comment/doc to explain why it is still there? Otherwise, someone might delete it as it's not doing anything.

Reply:
I'm dropping the field